### ISSUE TITLE
Large mbox file not restored to Gmail account

### ISSUE THREAD

**Issue by FachryMyID** [jay0lee/got-your-back#155]
I want restore large MBOX file 3GB while I get from Google Takeout, but nothing happen when I running all command right.

Full steps to reproduce the issue:
1. gyb --email xxxxx@gmail.com --action restore-mbox --local-folder "c:\my_gmail_backup"
2. Only appear message Using backup folder c:\my_gmail_baackup
3. Nothing

Expected outcome (what are you trying to do?):
Restored all mbox file to Gmail
Actual outcome (what errors or bad behavior do you see instead?): Nothing, see screenshot https://lh3.googleusercontent.com/-AQpxbLsj1Kk/WfqCtI-YPRI/AAAAAAAA3NY/Sik4hwHNdG0ivNcBcqjuJzBo1-1tl1GzQCLcBGAs/s1600/Untitled.png

**Comment by peterlert** [comment]
Sadly, I am seeing the same behavior with a small mbox file. I'm running on MacOS. Any suggestions would be greatly appreciated. Thanks for your work!

**Comment by newdawnguitar** [comment]
Dug into this a bit and figured out the problem. OSX is interpreting any .mbox file as a directory, and not a file. 

For my particular mbox that I exported, within the .mbox file/directory is another file simply called "mbox" (no extension). This was a result of an export from Apple mail, so ymmv depending on which app your exported your mailbox from.

Anyway, in my particular case, I changed this line of code in gyb.py to look for the mbox file within the .mbox subdirectory. Line 1248 and 1249:

`if filename[-3:].lower() != 'mbx' and \
          filename[-4:].lower() != 'mbox':`

This simply checks for a file ending in mbox and not an extension ending in mbox. So double check your files before using this hack.
